plugins/modules/aci_config_rollback.py

```diff
@@ -291,17 +293,32 @@ def get_preview(aci):
     This function is used to generate a preview between two snapshots and add the parsed results to the aci module return data.
     '''
     uri = aci.url + aci.filter_string
-    resp, info = fetch_url(aci.module, uri, headers=aci.headers, method='GET', timeout=aci.module.params.get('timeout'),
-                           use_proxy=aci.module.params.get('use_proxy'))
+    path = aci.path + aci.filter_string
+    resp = None
+    if aci.module._socket_path:
+        connect = Connection(aci.module._socket_path)
+        connect.set_params(aci.headers.get('Cookie'), aci.params)
+        info = connect.send_request('GET', '/{0}'.format(path), None)
+        aci.url = info.get('url')
+        aci.httpapi_logs.extend(connect.pop_messages())
+    else:
+        resp, info = fetch_url(aci.module, uri, headers=aci.headers, method='GET', timeout=aci.module.params.get('timeout'),
+                               use_proxy=aci.module.params.get('use_proxy'))
```

Comment:
why not using the same method as everything else? Why redefining it here?

Reply:
I added the changes here when api_call() was not successful when it was called from here.